.github/workflows/build_and_test_library.yml

```diff
@@ -125,4 +125,4 @@ jobs:
       - name: "Release to GitHub"
         uses: softprops/action-gh-release@v1
         with:
-          files: ~/dist/*.whl
+          files: ~/dist/ansys-grantami-serverapi-openapi-wheel/*.whl
```

Comment:
Artifacts are uploaded with a `name` arg, so they get downloaded to a folder with said name